doc/source/reference/python_introduction.rst

```diff
@@ -643,7 +637,7 @@ covers mechanisms for combining arrays of different shapes and sizes, which can
 be tricky but also extremely powerful.  We won't discuss the idea of
 broadcasting here, simply because I don't know that I could do it justice!  The
 NumPy Docs have a great `section on broadcasting
-<https://docs.scipy.org/doc/numpy/user/basics.broadcasting.html>`_.
+<https://numpy.org/doc/stable/user/basics.broadcasting.html>`_.
```

Comment:
good catch ! While you're at it, can you update the remaining link to docs.scipy...numpy ? It's in doc/source/conf.py :)

Reply:
Yep